pkg/networkservice/chains/nsmgr/unix_test.go

```diff
@@ -38,6 +38,10 @@ import (
 	"github.com/networkservicemesh/sdk/pkg/tools/sandbox"
 )
 
+const (
+	linuxOsName = "linux"
+)
+
```

Comment:
```suggestion
```

Reply:
This change was required by `golangci-lint`

Reply:
Just don't check on OS name. See at first comment.

Reply:
Right. Thanks, Denis!